tests/pos-macros/i8302.scala

```diff
@@ -1,6 +1,6 @@
 import scala.quoted._
 def foo[T](using qctx: QuoteContext, tpe: Type[T]): Expr[Any] =
-  '{
+  '{ (qctx: QuoteContext) ?=>
```

Comment:
Is this necessary? BTW, the recommended syntax is `(using QuoteContext) => e` if I remember correctly.

Reply:
Yes, that is the quote context for the next stage. Similar reason as the previous one. Will change the syntax.

Reply:
Updated